packages/design-tokens/src/tokens/functional/components/faq/colors.json

```diff
@@ -1,10 +1,28 @@
 {
   "brand": {
     "FAQ": {
-      "heading": {
-        "color": {
+      "color": {
+        "heading": {
           "value": "var(--brand-color-text-default)",
           "dark": "var(--brand-color-text-default)"
+        },
+        "tabBorder": {
+          "selected": {
+            "value": "hsl(var(--base-color-scale-blue-5-hsl) / 20%)",
+            "dark": "hsl(var(--base-color-scale-blue-4-hsl) / 20%)"
+          }
+        },
+        "tabLabel": {
+          "selected": {
+            "value": "var(--brand-Link-color-accent)",
+            "dark": "var(--brand-Link-color-accent)"
+          }
+        },
+        "tabBg": {
+          "selected": {
+            "value": "hsl(var(--base-color-scale-blue-5-hsl) / 08%)",
```

Comment:
In pro of using solid colors, could this be --base-color-scale-blue-0?  cc @jesussandreas 

Using transparency over that shade of blue causes the color to be a bit more greyish than the normal saturated blue


<table>
<tr>
<th> Blue 5 with transparency </th> <th> Solid blue 0 </th>
</tr>
<tr>
<td valign="top">

<img width="765" alt="Screenshot 2023-09-01 at 10 19 57" src="https://github.com/primer/brand/assets/912236/0a3e3c32-c527-4393-890b-3a29fdffcc68">


 </td>
<td valign="top">

<img width="758" alt="Screenshot 2023-09-01 at 10 19 40" src="https://github.com/primer/brand/assets/912236/ef69fab2-30d5-41d0-9f11-5f87a8729750">


</td>
</tr>
</table>

Reply:
yeah, that works!